ui/apps/platform/src/Components/CompoundSearchFilter/components/SearchFilterAutocomplete.tsx

```diff
@@ -0,0 +1,288 @@
+import React, { useMemo, useRef, useState } from 'react';
+import {
+    Select,
+    SelectOption,
+    SelectList,
+    SelectOptionProps,
+    MenuToggle,
+    MenuToggleElement,
+    TextInputGroup,
+    TextInputGroupMain,
+    TextInputGroupUtilities,
+    Button,
+    Skeleton,
+    Flex,
+    debounce,
+} from '@patternfly/react-core';
+import TimesIcon from '@patternfly/react-icons/dist/esm/icons/times-icon';
+import { ArrowRightIcon } from '@patternfly/react-icons';
+import { useQuery } from '@apollo/client';
+import SEARCH_AUTOCOMPLETE_QUERY, {
+    SearchAutocompleteQueryResponse,
+} from 'queries/searchAutocomplete';
+
+type SearchFilterAutocompleteProps = {
+    searchCategory: string;
+    searchTerm: string;
+    value: string;
+    onChange: (value: string) => void;
+    onSearch: (value: string) => void;
+    textLabel: string;
+};
+
+function getSelectOptions(
+    data: SearchAutocompleteQueryResponse | undefined,
+    isLoading: boolean
+): SelectOptionProps[] {
+    if (isLoading) {
+        return [
+            {
+                isDisabled: true,
+                value: 'autocomplete-options-loading',
+                children: (
+                    <Flex
+                        direction={{ default: 'column' }}
+                        spaceItems={{ default: 'spaceItemsMd' }}
+                    >
+                        <Skeleton screenreaderText="Loading suggested options 1" width="100%" />
+                        <Skeleton screenreaderText="Loading suggested options 2" width="75%" />
+                        <Skeleton screenreaderText="Loading suggested options 3" width="90%" />
```

Comment:
I see your point. I guess the `options X` isn't as helpful. It would have been nice if we could do a `SkeletonGroup` and put the screen reader text on that. It doesn't seem like PF has that option. I can do a single skeleton, but I felt like it looked nicer to have a few to show a list of things. Maybe I'll ask UX their thoughts too